cms/bip/layouts/homepage/content.html

```diff
@@ -3,9 +3,7 @@
     {{ cms:page:hero_text:rich_text }}
```

Comment:
can we include also here the hero_title block please? We got the CMS fixtures enabled, meaning that changing this file should automagically update the cms structure (this is not working on my local, though) so it would be good to record any changes to CMS layout and pages here as well

Reply:
Added in new commit